images/fcos-auxiliary-image/root/usr/bin/libvirt_deploy.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+libvirt=`yum list installed | grep libvirt`
+
+# 'dnf -y install bridge-utils libvirt virt-install qemu-kvm" if libvirtd is not installed
+if [[ -z ${libvirt} ]]; then
+    dnf -y install bridge-utils libvirt virt-install qemu-kvm
+else
+# Start libvirtd if it is inactive
+    is_active=`systemctl is-active libvirtd`
+    if [ "$status" = "inactive" ]; then
+        systemctl start libvirtd
+    fi
+# Enable libvirtd if it is disabled
+    is_enabled=`systemctl is-enabled libvirtd`
+    if [ "$status" = "disabled" ]; then
+        systemctl enable libvirtd
+    fi
+fi
```

Comment:
we do not need all this snipped as we can ensure that via the layered image and by setting libvirt enabled by default.